index.js

```diff
@@ -4,29 +4,30 @@
 var ncp = require('copy-paste');
 var ngrok = require('ngrok');
 var Promise = require('ember-cli/lib/ext/promise');
+var ServeCommand = require('ember-cli/lib/commands/serve');
 
 module.exports = {
   name: 'ember-share',
   includedCommands: function() {
     return {
-      share: {
+      share: ServeCommand.extend({
         name: 'share',
         description: 'Share your local Ember apps with the world using ngrok.',
-        availableOptions: [
```

Comment:
ember share now behaves like `serve`, it takes all the available options from the ServeCommand